pages/linux/ipcmk.md

```diff
@@ -0,0 +1,19 @@
+# ipcmk
+
+> Create IPC (Inter-process Communication) resources.
+
+- Create a shared memory segment:
+
+`ipcmk --shmem {{segment_size_in_bytes}}`
+
+- Create a semaphore:
+
+`ipcmk --semaphore {{element_size}}`
+
+- Create a message queue:
+
+`ipcmk --queue`
+
+- Create a shared memory segment with permissions other than 0644:
```

Comment:
I think it might be better to say - "Create a shared memory segment with specific permissions (default is 0644)."

Atleast that's what I understand.

Reply:
I prefer this phrasing as well, will patch when I'm at a computer.

Thank you!